package.json

```diff
@@ -8,8 +8,8 @@
   },
   "homepage": "https://react-bootstrap.github.io/",
   "sideEffects": false,
-  "main": "lib/index.js",
-  "module": "lib/es/index.js",
+  "main": "lib/cjs/index.js",
```

Comment:
I think this should be `cjs/index.js` and the following line is `esm/index.js`

Reply:
our build process strips the prefix